js.go

```diff
@@ -718,6 +718,10 @@ func (m *Msg) checkReply() (*js, bool, error) {
 	}
 	sub := m.Sub
 	sub.mu.Lock()
+	if sub.jsi == nil {
+		sub.mu.Unlock()
+		return nil, false, ErrNotJSMessage
+	}
```

Comment:
Let's remove the check for `js == nil` down there..

Reply:
ok just saw this and already merged but will do so.